tests/rapids_pre_commit_hooks/utils/test_yaml.py

```
@@ -79,9 +75,32 @@ def test_check_and_mark_anchor(
     used_anchors = set(used_anchors_before)
-    actual_descend, actual_anchor = check_and_mark_anchor(
+    actual_anchor = check_and_mark_anchor(
         ANCHORS, used_anchors, NODES[node_index]
     )
-    assert actual_descend == descend
     assert actual_anchor == anchor
     assert used_anchors == used_anchors_after
+
+
+@pytest.mark.parametrize(
+    ["anchor", "is_ref"],
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)